src/components/Layout/Header/ActionCenter/components/Details/StreamingSwapDetails.tsx

```diff
@@ -53,9 +63,11 @@ export const StreamingSwapDetails: React.FC<StreamingSwapDetailsProps> = ({ swap
           size='xs'
           value={progress}
           colorScheme={isComplete ? 'green' : 'blue'}
+          isAnimated={true}
+          hasStripe={isComplete ? false : true}
```

Comment:
Same comment here as in https://github.com/shapeshift/web/pull/9695/files#r2150396035 re: are we properly decoupling swap status and streaming status? Doesn't look like it, meaning this will stop animating once the streaming part of the swap is complete, but wondering if it wouldn't be more natural to keep it animated until complete?
Also, wondering if we want to add `getLatestThorTxStatusMessage()` (and equivalent for Chainflip) similar to swapper here, to make those actions more useful/less confusing?
As a user, seeing progress with the same human-readable parsing of status we're doing in swapper would be a game-changer, especially in that specific case of discrepancies between streaming status and swap status 🙏🏽

Reply:
Used the swap status only in [53df27f](https://github.com/shapeshift/web/pull/9695/commits/53df27f350df1a851f51947e6180239eb47782f0)

I'll leave `getLatestThorTxStatusMessage` for now and see with product how we can show up this text